### PR TITLE
get scala-xml working in 2.11.x-jdk8 again

### LINKED ISSUE #538: regression: Scaladoc blows up in 2.11.x-jdk8 build after scala-xml changes
last green build was scala/scala-xml@383f5fa34335c156e077dc93fc0c7cf40f8d0ec5 https://scala-ci.typesafe.com/view/scala-2.11.x/job/scala-2.11.x-jdk8-integrate-community-build/372/

first failing build scala/scala-xml@8495b63b132593d96f55876a4bc050578bed22db https://scala-ci.typesafe.com/view/scala-2.11.x/job/scala-2.11.x-jdk8-integrate-community-build/373/

scala-xml itself builds, but then multiple projects fail during Scaladoc generation, e.g.

```
[genjavadoc-plugin] java.lang.NoSuchMethodError: scala.collection.immutable.Traversable.$init$(Lscala/collection/immutable/Traversable;)V
[genjavadoc-plugin] 	at scala.xml.NodeSeq.<init>(NodeSeq.scala:45)
[genjavadoc-plugin] 	at scala.xml.Node.<init>(Node.scala:36)
[genjavadoc-plugin] 	at scala.xml.SpecialNode.<init>(SpecialNode.scala:21)
[genjavadoc-plugin] 	at scala.xml.Atom.<init>(Atom.scala:19)
[genjavadoc-plugin] 	at scala.xml.Text.<init>(Text.scala:19)
[genjavadoc-plugin] 	at scala.tools.nsc.doc.html.page.Index.<init>(Index.scala:26)
[genjavadoc-plugin] 	at scala.tools.nsc.doc.html.HtmlFactory.generate(HtmlFactory.scala:124)
```

relevant history in scala-xml repo:

```
*   8495b63 - (HEAD -> master, origin/master, origin/HEAD) Merge pull request #126 from ashawley/enable-test-forking (3 days ago) <Aaron S. Hawley>
|\  
| * 006afa4 - Fix exclude rule for scala-xml from scala-compiler (3 days ago) <Aaron S. Hawley>
|/  
*   121034f - Merge pull request #145 from lrytz/tag-with-scala-version (3 days ago) <Aaron S. Hawley>
|\  
| * 66656aa - Allow publishing existing tags against new Scala versions (3 days ago) <Lukas Rytz>
```

I'll look into whether this is somehow just because we're in dbuild, or whether this is a real problem in the scala-xml repo

fyi @ashawley

### PR DESCRIPTION
**Pull request by SethTisue**
fixes #538

### CODE REVIEW

**Comment by SethTisue** [comment]
short run showing a previously failing dependent project now succeeding: https://scala-ci.typesafe.com/view/scala-2.11.x/job/scala-2.11.x-jdk8-integrate-community-build/378/consoleFull

**Comment by SethTisue** [comment]
whoops... fixup in d97bb1594c5b7ccdae26810e68bfec41066aaa35

File: common.conf [hunk 1]
```diff
@@ -76,7 +76,7 @@ vars: {
   scala-parser-combinators-ref : "scala/scala-parser-combinators.git"
   scala-partest-ref            : "scala/scala-partest.git#1.0.x"
   scala-partest-interface-ref  : "scala/scala-partest-interface.git"
-  scala-xml-ref                : "scala/scala-xml.git"
+  scala-xml-ref                : "scala/scala-xml.git#121034f"
```
Comment (lrytz) [inline common.conf:79]: Is this intentional or accidental?

Comment (SethTisue) [reply common.conf:79]: accidental, fixed in the fixup commit

**Pull request merged**